Installers/Electron/src/utils.js

```diff
@@ -73,8 +73,8 @@ class Utils {
                 });
                 return path;
             },
-            "darwin": () => "/Applications/Discord.app",
-            "linux": () => "" // TODO
+            "darwin": () => "/Applications/Discord.app/Contents",
+            "linux": () => "/usr/share/discord-canary"
```

Comment:
I added this knowing that it was definitely not used everywhere, but it is the path used by the official .deb release, making it a reasonable default. The ideal solution would be to automatically scan through the likely install locations, especially now that the stable release is out.